client/controllers/incidentReports/incidentReport.coffee

```diff
@@ -12,3 +17,10 @@ Template.incidentReport.helpers
 
   importantDetails: ->
     @deaths or @cases or @status
+
+  incidentUrl: ->
+    Articles.findOne(@articleId)?.url
+
+  incidentContent: ->
+    articleContent = Articles.findOne(@articleId).enhancements.source.cleanContent.content
+    Spacebars.SafeString(getIncidentSnippet(articleContent, @, 100))
```

Comment:
Since this is currently the only call site, it would make sense to set the default padding characters value to 100.